### ISSUE TITLE
Missing Window Icon + not sure how to play

### ISSUE THREAD

**Issue by MikeTheWatchGuy** [allenfrostline/Poker#3]
When I see the window, I see the default tkinter icon.

After I see this screen, I don't seem to be able to do anything, at least nothing I click seems to do anything.  I'll read through the code a bit and see what I'm missing.

![image](https://user-images.githubusercontent.com/13696193/56304718-ba9f1380-610c-11e9-9447-0a22bc8b75fd.png)

**Comment by allenfrostline** [comment]
Well that's strange. Perhaps something to do with Windows. Thanks for help though :)

**Comment by MikeTheWatchGuy** [comment]
There is not poker.icns file in the resource folder on GitHub, thus it won't be loading anything as the icon as shown in this statement:
```
        window = sg.Window(window_title, icon=absPath('resources/poker.icns'), size=(564, 270 + 40 * (n_players - 2)), disable_close=True,
```

**Comment by allenfrostline** [comment]
Fixed.

**Comment by MikeTheWatchGuy** [comment]
Hmmm... I don't see it in the GitHub folder that I'm looking at (and the code from what I can tell):

![image](https://user-images.githubusercontent.com/13696193/56305195-cb03be00-610d-11e9-969a-504eb4da0e31.png)

## Also, is there something that I can click on, etc, to play the game or is this missing icon causing problems blocking me?

**Comment by allenfrostline** [comment]
Weird... I see the `icns` file uploaded to root dir of this repo on my computer. Perhaps a refresh of the page will help?

The game view you just posted above is supposed to be, when on a Mac, just like the screenshot you can see in README. So there will be several buttons.

**Comment by MikeTheWatchGuy** [comment]
I see, you uploaded to root, but the code expects in resources so I copied there.

It didn't matter in the end as .icns is not a valid file format for windows.  Can you supply it as a PNG or a GIF?  This will allow it to go cross-platform.  Or, perhaps better yet, making it into a base64 variable that is a permanent part of your code (assuming it's in the right format).

Do you see the default PySimpleGUI Icon when you run the program without setting your icon?

**Comment by MikeTheWatchGuy** [comment]
Oh, I see the diff now with the readme... there are no buttons on the Windows version.

**Comment by MikeTheWatchGuy** [comment]
The problem was that you're manually setting the size of your window.  This had the effect of slicing off the bottom portion of the window.

Something about that calculation seems off.  Your Output element looks crazy too with a size of 1150,5.  That's 1150 characters wide.

**Comment by MikeTheWatchGuy** [comment]
After changing the Output Element size and the Text size for the "Game 1", the window "naturally sized" to the size it wanted to be which was this:
![image](https://user-images.githubusercontent.com/13696193/56306247-fd161f80-610f-11e9-9145-d1606b8eed98.png)

So, it's closer.

**Comment by MikeTheWatchGuy** [comment]
With a border width of zero and no title, I don't understand the purpose of the Frame Element that's being used with the buttons.  You can simply put the buttons on the same row and get the same effect.  You're doing a lot of manual padding of the buttons anyway.  That's what's pushing the "Call button" over which pushes everything else.

**Comment by allenfrostline** [comment]
Exactly. I've been spending a lot of time with all those padding... I'm still not quite familiar with all those units of element sizes (some are in pixels and the rest in characters).

**Comment by allenfrostline** [comment]
Also, it's strange as the window shows well on Mac. The buttons were fine there.

**Comment by MikeTheWatchGuy** [comment]
Element sizes are pretty much always in Characters, Rows.

Padding is in Pixels

Some ports of PySimpleGUI you can specify both in the size field, but not in the tkinter one.  Newer ports have a size_px parameter if you want to specify the size in pixels, but again, not something in tkinter version.
